javascript-conventions.md

```diff
@@ -1,32 +1,76 @@
 # JavaScript codeerstandaarden en naamgevingsconventies
 
-## 1. Gebruik van `var`, `let` en `const` zoals hieronder beschreven.
-De scope van een variabele kan **lokaal** of **globaal** zijn. Globale variabelen kunnen op willekeurige plaatsen in het programma worden aangeroepen. Lokale variabelen worden in een functie of een codeblok gedeclareerd. Hun bereik beperkt zich tot deze functie of
-codeblok.
-
-Voor de duidelijkheid maken we de volgende afspraken:
-* `var` gebruiken we enkel voor variabelen die globaal gebruikt worden, doorheen gans je script dus.
- * `let` gebruiken we enkel voor variabelen die binnen een bepaalde function of code block (if, lus …) gebruikt wordt
-* `const` behouden we voor waarden die nooit gewijzigd worden
-
-|| `var` | `let` | `const` |
-|---|---|---|---|
-|gedeclareerd **buiten** een function| globaal | globaal | globaal |
-|gedeclareerd **buiten** een code block `{ ... }` | globaal | globaal | globaal |
-|gedeclareerd **binnen** een code block `{ ... }`| globaal | lokaal | lokaal |
-|gedeclareerd **binnen** een function| lokaal | lokaal | lokaal |
+## 1. Gebruik `let`, `const` en `var` zoals hieronder beschreven.
+Wanneer je een variabele of constante declareert, gebruik dan `let` of `const` en *niét* `var`.
+
+Wanneer een variable geen nieuwe waarde toegekend zal krijgen, verkies dan `const`:
+```javascript
+const userName = "Chris";
+console.log(userName);
+```
+
+Zoniet, gebruik dan `let`:
+```javascript
+let age = 40;
+age++;
+console.log(age);
+```
+
+Het voorbeeld hieronder toont het gebruik van `let`, terwijl er beter gekozen werd voor `const`. Deze code werkt, maar dient vermeden te worden:
+```javascript
+//fout
+let userName = "Chris";
+console.log(userName);
+```
+
+Het voorbeeld hieronder betreft het gebruik van `const`, terwijl het een nieuwe deze een nieuwe waarde toegekend krijgt. De toekenning van de nieuwe waarde
+gooit een error op:
+```javascript
+//fout
+const age = 40;
+age++;
+console.log("Happy Birthday!");
+```
+
+Dit voorbeeld gebruikt `var`. Vermijd `var` ten allen tijd, tenzij je het echt nodig is:
```

Comment:
Nog nodig? `var` is immers niet meer toegelaten is. Wanneer zou het dan **echt** nodig zijn? 
evt voorbeeld laten maar 
> tenzij je het echt nodig is

eruit halen. Bevat overigens grammaticafoutje :)